### PR TITLE
Mongo, Lambdas and S3 utility functions

### PR DESCRIPTION
**Pull request by graymur**
Set of utility functions, here's a readme: https://github.com/Apination/cn-utils/blob/dev/README.md

### CODE REVIEW

**Comment by Restuta** [comment]
Please add repository description.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+## Utility function to be used in connectors
+
+### Lambda functions
+
+#### `const validateAndExecute = require('cn-utils').lambdaValidateAndExecute`
```
Comment (Restuta) [inline README.md]: How about namespasing this furhter:

`const laValidateAndExecute = require('cn-utils/lambdas').validateAndExecute`

which implies `/lambdas` folder with `index.js` and proper exports, or

`const laValidateAndExecute = require('cn-utils/lambdas/validate-and-execute')`

with no index.js

**Comment by Restuta** [comment]
Also why is this high priority?

**Comment by graymur** [comment]
@Restuta because I need this in other lambdas. I'm already using in from #dev branch, specifically `uploadToS3`

**Comment by graymur** [comment]
@Restuta Reorganized file structure.

**Pull request merged**